api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -250,7 +250,14 @@ message Bootstrap {
   // Setting this value causes failure if the
   // ``envoy.restart_features.use_apple_api_for_dns_lookups`` runtime value is true during
   // server startup. Apple' API only uses UDP for DNS resolution.
-  bool use_tcp_for_dns_lookups = 20;
+  // This field is deprecated. Instead use
+  // :ref:`dns_lookup_options <envoy_api_field_config.config.bootstrap.v3.Bootstrap.dns_lookup_options>`
```

Comment:
I think there's a wrong link here. Should probably be:
`envoy_api_field_config.bootstrap.v3.Bootstrap.dns_lookup_options`.
Please fix here and in other places.

Reply:
Thanks for taking a look. I am working on these changes on the side. I will try to push rest of the changes by tomorrow. I will fix this :ref: link as well.